docs/spec/blockchain/encoding.md

```diff
@@ -0,0 +1,346 @@
+# Encoding
```

Comment:
This file is obsolete, so should be removed from this PR. The up-to-date version is `spec/core/encoding.md`

Reply:
Resolved